library/flink

```diff
@@ -4,32 +4,42 @@ Maintainers: Patrick Lucas <me@patricklucas.com> (@patricklucas),
              Ismaël Mejía <iemejia@gmail.com> (@iemejia)
 GitRepo: https://github.com/apache/flink-docker.git
 
-Tags: 1.11.2-scala_2.12-java8, 1.11-scala_2.12-java8, scala_2.12-java8, 1.11.2-scala_2.12, 1.11-scala_2.12, scala_2.12, 1.11.2-java8, 1.11-java8, java8, 1.11.2, 1.11, latest
+Tags: 1.11.2-scala_2.11-java11, 1.11-scala_2.11-java11
 Architectures: amd64
 GitCommit: 149ebb09d9146ec46c6df0ab73d638c4d7549c10
-Directory: ./1.11/scala_2.12-java8-debian
+Directory: ./1.11/scala_2.11-java11-debian
 
-Tags: 1.11.2-scala_2.11-java8, 1.11-scala_2.11-java8, scala_2.11-java8, 1.11.2-scala_2.11, 1.11-scala_2.11, scala_2.11
+Tags: 1.11.2-scala_2.11-java8, 1.11-scala_2.11-java8, 1.11.2-scala_2.11, 1.11-scala_2.11
 Architectures: amd64
 GitCommit: 149ebb09d9146ec46c6df0ab73d638c4d7549c10
 Directory: ./1.11/scala_2.11-java8-debian
 
-Tags: 1.11.2-scala_2.12-java11, 1.11-scala_2.12-java11, scala_2.12-java11, 1.11.2-java11, 1.11-java11, java11
+Tags: 1.11.2-scala_2.12-java11, 1.11-scala_2.12-java11, 1.11.2-java11, 1.11-java11
 Architectures: amd64
 GitCommit: 149ebb09d9146ec46c6df0ab73d638c4d7549c10
 Directory: ./1.11/scala_2.12-java11-debian
 
-Tags: 1.11.2-scala_2.11-java11, 1.11-scala_2.11-java11, scala_2.11-java11
+Tags: 1.11.2-scala_2.12-java8, 1.11-scala_2.12-java8, 1.11.2-scala_2.12, 1.11-scala_2.12, 1.11.2-java8, 1.11-java8, 1.11.2, 1.11
 Architectures: amd64
 GitCommit: 149ebb09d9146ec46c6df0ab73d638c4d7549c10
-Directory: ./1.11/scala_2.11-java11-debian
+Directory: ./1.11/scala_2.12-java8-debian
+
+Tags: 1.12.0-scala_2.11-java11, 1.12-scala_2.11-java11, scala_2.11-java11
+Architectures: amd64
+GitCommit: 041ac78d9264cc596d9007eb524d24c2b65ca830
+Directory: ./1.12/scala_2.11-java11-debian
+
+Tags: 1.12.0-scala_2.11-java8, 1.12-scala_2.11-java8, scala_2.11-java8, 1.12.0-scala_2.11, 1.12-scala_2.11, scala_2.11
+Architectures: amd64
+GitCommit: 041ac78d9264cc596d9007eb524d24c2b65ca830
+Directory: ./1.12/scala_2.11-java8-debian
 
-Tags: 1.10.2-scala_2.12, 1.10-scala_2.12, 1.10.2, 1.10
+Tags: 1.12.0-scala_2.12-java11, 1.12-scala_2.12-java11, scala_2.12-java11, 1.12.0-java11, 1.12-java11, java11
 Architectures: amd64
-GitCommit: 58a29fca7c6ff05999fad4371638d16335f7e93e
-Directory: ./1.10/scala_2.12-debian
+GitCommit: 041ac78d9264cc596d9007eb524d24c2b65ca830
+Directory: ./1.12/scala_2.12-java11-debian
 
-Tags: 1.10.2-scala_2.11, 1.10-scala_2.11
+Tags: 1.12.0-scala_2.12-java8, 1.12-scala_2.12-java8, scala_2.12-java8, 1.12.0-scala_2.12, 1.12-scala_2.12, scala_2.12, 1.12.0-java8, 1.12-java8, java8, 1.12.0, 1.12, latest
```

Comment:
What is the reason to allow java8 images?

Reply:
Java 8 is the lowest version required and the maven target version for Flink, and is AFAIK used by the majority of Flink developers and users.